app/controllers/journeys/adjustments/profitOrLoss/ProfitOrLossCalculationController.scala

```diff
@@ -42,12 +44,26 @@ class ProfitOrLossCalculationController @Inject() (override val messagesApi: Mes
     with Logging {
 
   def onPageLoad(taxYear: TaxYear, businessId: BusinessId): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    val summaryList = SummaryListCYA.summaryListOpt(List())
+    val summaryList               = SummaryListCYA.summaryListOpt(List())
```

Comment:
Looks like summary list isn't doing anything in the controller or the view, can be removed?